src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/JSON1006.ValueTooLarge_LocationId_Invalid.sarif

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
```

Comment:
>tool [](http://example.com/codeflow?start=7&length=4)

It is very inefficient to add 8 log files to our tests for this specific bug fix. Please create a pattern in the unit tests for validating this problem.

Reply:
yes, will change to only one 1 input file for each rule according to your other comment